tests/src/test/java/alluxio/hadoop/HdfsFileInputStreamIntegrationTest.java

```diff
@@ -67,11 +74,12 @@ public final void after() throws Exception {
 
   @Before
   public final void before() throws IOException, AlluxioException {
-    AlluxioURI file1 = new AlluxioURI("/testFile1");
-    mInMemInputStream = new HdfsFileInputStream(file1, new Configuration(), BUFFER_SIZE, null);
+    mFile1 = new AlluxioURI("/testFile1");
+    mInMemInputStream = new HdfsFileInputStream(mFile1, new Configuration(), BUFFER_SIZE, null);
 
-    AlluxioURI file2 = new AlluxioURI("/testFile2");
-    mUfsInputStream = new HdfsFileInputStream(file2, new Configuration(), BUFFER_SIZE, null);
+    ClientContext.getConf().set(Constants.USER_FILE_READ_TYPE_DEFAULT, "NO_CACHE");
```

Comment:
should this happen in the tests you added instead of in the setup? Changing it here will affect all tests.